index.html

```diff
@@ -933,7 +933,7 @@ <h3>Concrete Lifecycle Example</h3>
     <span class='comment'>// purpose of this proof</span>
     "proofPurpose": "assertionMethod",
     <span class='comment'>// the identifier of the public key that can verify the signature</span>
-    "verificationMethod": "https://example.edu/issuers/565049/keys/1",
+    "verificationMethod": "https://example.edu/issuers/565049#key-1",
```

Comment:
How are you linking these particular verificationMethods? I was looking over the respec-vc implementation and didn't see you hard coding the private keys in to test this and from the looks of it the keys were being randomly generated on the fly so wasn't sure if people would be able to verify these or if that's acceptable for them to not be.

Reply:
The short of it is that we're not (yet). It's a future enhancement for sure.

I was thinking of adding a section in the appendix of the spec that lists the public/private keys used for all examples as well as the documents located at the verification method URLs. Since they're generated per page refresh, we could either 1) hard code some keys, or 2) be okay with using new keys for every spec. There are benefits and drawbacks for both approaches.

Reply:
Makes sense - good to know. I think I have a general idea of what is going on with that code now as well after this and think I could help to maintain it as we get specific requests from the WG on it in the future.

Reply:
I would love help maintaining that package. Specifically, I want to get a BBS+ signature in there to demonstrate that we're on top of the whole "general purpose solution to selective disclosure" problem space.